### PR TITLE
Node week2 main

### PR DESCRIPTION
**Pull request by FurkannOzbek**
* Meals route has been added (GET,PUT,DELETE and POST methods has been added)
 * Reservations route has been added (GET,PUT,DELETE and POST methods has been added)

### CODE REVIEW

File: api/src/routers/meals.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+meals.post("/", async (req, res) => {
+  const body = req.body;
+
+  if (body) {
+    await knex("Meal").insert(body);
+  }
+});
+
+meals.get("/:id", async (req, res) => {
+  const mealId = parseInt(req.params.id, 10);
```
Comment (magdazelena) [inline api/src/routers/meals.js:20]: oh, interesting. can you explain this line?

File: api/src/routers/meals.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const selectedMeal = await knex("Meal").where("id", mealId);
+  res.send(selectedMeal);
+});
+
+meals.put("/:id", async (req, res) => {
+  const body = req.body;
+  const mealId = parseInt(req.params.id, 10);
+  const selectedMeal = await knex("Meal").where("id", mealId);
+  if (body) {
```
Comment (magdazelena) [inline api/src/routers/meals.js:29]: probably some sort of parsing and validation would be nice before updateing or inserting anything into db. You never know if the api is getting the accurate data or not :)

**Review by magdazelena — COMMENTED** [review]
couple of comments, otherwise looking good :)

File: api/src/routers/meals.js (hunk 1 above)
Comment (FurkannOzbek) [reply api/src/routers/meals.js:20]: Oh, when I tried like `mealID  = req.params.id ` I wasnt able to get the parameter from the url. Thats why I have used I thought like maybe I needed to make it integer. But I have tried now as mealID = `req.params.id ` and it worked. So I probably got error of something else there :)

**Pull request merged**